source/configure/calls-deployment.rst

```diff
@@ -0,0 +1,347 @@
+
+Do calls require a dedicated server to work or can they run alongside Mattermost?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The plugin can function in different modes. By default calls are handled completely by the plugin which runs as part of Mattermost. It's also possible to use a dedicated service to offload the computational and bandwidth costs and scale further (Enterprise only).
+
+Can the traffic between Mattermost and ``rtcd``  be kept internal or should it be opened to the public?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When possible, it's recommended to keep communication between the Mattermost cluster and the dedicated ``rtcd`` service under the same private network as this can greatly simplify deployment and security. There's no requirement to expose ``rtcd``'s HTTP API to the public internet.
+
+Debugging
+---------
+
+If you experience issues with calls, collecting information is helpful as you can share it with us for debugging purposes. 
+
+As with any other issue, but more importantly with calls, it’s very useful to provide date and time of when some problem occurred, with as much detail as possible so that information can be cross-checked with server logs as well. Also please include any reproduction steps if applicable. Other important information include:
+
+- Browser/app version
+- Operating system type and version
+
+JS console logs
+~~~~~~~~~~~~~~~
+
+Web app
+^^^^^^^
+
+- **Chrome**: CMD+OPTION+J (macOS)/CTRL+SHIFT+J (Windows, Linux, ChromeOS)
+- **Firefox**: CMD+SHIFT+J (macOS)/CTRL+SHIFT+J (Windows, Linux, ChromeOS).
+- **Safari**: Enable Developer Menu in **Safari > Preferences > Advanced > Show Develop Menu in Menu Bar**. Then **Develop > Show Javascript Console**. Right-click on the console and select **Save to file** to download the logs.
+
+Desktop app
+^^^^^^^^^^^
+
+In the top menu bar of the app, select **View > Developer Tools for Current Tab**. In the logs that are generated, right-click and select **Save as** to download the logs.
+
+Call stats dump
+---------------
+
+In cases where there are audio/video issues, difficulty in hearing other participants, and/or stuttering video and/or choppy audio, run the ``/call stats`` slash command in the channel where the call is currently active. This returns a JSON object via an ephemeral message.
+
+As this command only works while a call is active, this information needs to be collected before leaving a call.
```

Comment:
This restriction has been lifted. It's now possible to run the command after leaving the call. The only caveat is that we will only save data for the last joined call so joining again will wipe the previous one.